src/test/resources/testHw9.properties

```diff
@@ -0,0 +1,3 @@
+BASE_URL = https://api.trello.com/1/
+DEFAULT_BOARD_NAME = DefaultBoard
```

Comment:
do not store data in test.property/
create separate class in test or main for data storage